### ISSUE TITLE
“TPE using * trials with best loss” log

### ISSUE THREAD

**Issue by GeorgeW9527** [microsoft/nni#4227]
**Describe the issue**:
dispatcher.log中的打印是否有歧义？
我把(nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results的打印等级改成了info，观察下面的log可以发现。
“TPE using 2/2 trials with best loss”后面跟着的是val acc，而且是到目前epoch为止最低的val acc。

**Environment**:
- NNI version:2.4
- Training service (local|remote|pai|aml|etc):local
- Client OS:ubuntu
- Server OS (for remote mode only):
- Python version:3.7
- PyTorch/TensorFlow version:pytorch1.9
- Is conda/virtualenv/venv used?:conda
- Is running in Docker?:no

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
```
[2021-09-28 11:00:34] INFO (nni.experiment/MainThread) Creating experiment, Experiment ID: [36m5galkq4p[0m
[2021-09-28 11:00:34] INFO (nni.experiment/MainThread) Connecting IPC pipe...
[2021-09-28 11:00:34] INFO (nni.experiment/MainThread) Starting web server...
[2021-09-28 11:00:35] INFO (nni.experiment/MainThread) Setting up...
[2021-09-28 11:00:35] INFO (nni.runtime.msg_dispatcher_base/Thread-3) Dispatcher started
[2021-09-28 11:00:35] INFO (nni.retiarii.experiment.pytorch/MainThread) Web UI URLs: [36mhttp://127.0.0.1:8097 http://169.254.232.130:8097 http://10.38.232.130:8097[0m
[2021-09-28 11:00:38] INFO (nni.retiarii.experiment.pytorch/MainThread) Start strategy...
[2021-09-28 11:00:40] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) TPE strategy has been started.
[2021-09-28 11:00:40] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.027790 seconds
[2021-09-28 11:00:40] INFO (hyperopt.tpe/MainThread) TPE using 0 trials
[2021-09-28 11:00:42] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.030114 seconds
[2021-09-28 11:00:42] INFO (hyperopt.tpe/MainThread) TPE using 0 trials
[2021-09-28 12:37:16] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 12:37:16] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 12:37:16] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 12:37:17] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.032026 seconds
[2021-09-28 12:37:17] INFO (hyperopt.tpe/MainThread) TPE using 0 trials
[2021-09-28 12:37:21] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 0, 0.7767999768257141
[2021-09-28 13:59:20] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 13:59:20] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 13:59:20] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 13:59:21] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.036802 seconds
[2021-09-28 13:59:21] INFO (hyperopt.tpe/MainThread) TPE using 1/1 trials with best loss 0.776800
[2021-09-28 13:59:25] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 1, 0.7490000128746033
[2021-09-28 15:25:35] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 15:25:35] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 15:25:35] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 15:25:36] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.073132 seconds
[2021-09-28 15:25:36] INFO (hyperopt.tpe/MainThread) TPE using 2/2 trials with best loss 0.749000
[2021-09-28 15:25:40] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 2, 0.7773000001907349
[2021-09-28 16:52:29] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 16:52:29] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 16:52:29] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 16:52:31] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.032028 seconds
[2021-09-28 16:52:31] INFO (hyperopt.tpe/MainThread) TPE using 3/3 trials with best loss 0.749000
[2021-09-28 16:52:34] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 3, 0.7552000284194946
[2021-09-28 18:11:37] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 18:11:37] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 18:11:37] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 18:11:38] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.031462 seconds
[2021-09-28 18:11:38] INFO (hyperopt.tpe/MainThread) TPE using 4/4 trials with best loss 0.749000
[2021-09-28 18:11:41] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 4, 0.7505000233650208
[2021-09-28 19:38:46] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 19:38:46] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 19:38:46] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 19:38:46] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.029938 seconds
[2021-09-28 19:38:46] INFO (hyperopt.tpe/MainThread) TPE using 5/5 trials with best loss 0.749000
[2021-09-28 19:38:52] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 5, 0.7236999869346619
[2021-09-28 20:58:41] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 20:58:41] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 20:58:41] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 20:58:41] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.032691 seconds
[2021-09-28 20:58:41] INFO (hyperopt.tpe/MainThread) TPE using 6/6 trials with best loss 0.723700
[2021-09-28 20:58:43] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 6, 0.7896000146865845
[2021-09-28 22:19:48] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 22:19:48] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 22:19:48] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 22:19:50] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.031494 seconds
[2021-09-28 22:19:50] INFO (hyperopt.tpe/MainThread) TPE using 7/7 trials with best loss 0.723700
[2021-09-28 22:19:55] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 7, 0.714900016784668
[2021-09-28 23:39:11] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-28 23:39:11] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-28 23:39:11] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-28 23:39:11] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.033351 seconds
[2021-09-28 23:39:11] INFO (hyperopt.tpe/MainThread) TPE using 8/8 trials with best loss 0.714900
[2021-09-28 23:39:14] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 8, 0.7297999858856201
[2021-09-29 00:59:53] INFO (pytorch_lightning.utilities.distributed/Thread-2) GPU available: True, used: True
[2021-09-29 00:59:53] INFO (pytorch_lightning.utilities.distributed/Thread-2) TPU available: False, using: 0 TPU cores
[2021-09-29 00:59:53] INFO (pytorch_lightning.utilities.distributed/Thread-2) IPU available: False, using: 0 IPUs
[2021-09-29 00:59:54] INFO (hyperopt.tpe/MainThread) tpe_transform took 0.029238 seconds
[2021-09-29 00:59:54] INFO (hyperopt.tpe/MainThread) TPE using 9/9 trials with best loss 0.714900
[2021-09-29 00:59:56] INFO (nni.retiarii.strategy.tpe_strategy/MainThread) tpe receive results: 9, 0.7427999973297119
```

 - nnictl stdout and stderr:
 

**How to reproduce it?**:
